.github/workflows/ci.yml

```diff
@@ -60,12 +60,12 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: 20.x
-      - run: yarn workspaces focus hono-middleware @hono/${{ matrix.package }}
+      - run: yarn workspaces foreach --topological --recursive --from @hono/${{ matrix.package }} run build
```

Comment:
Specifically fixes workspaces that depend on other workspaces being built first, eg; `@hono/zod-validator` and `@hono/zod-openapi` 🙈